spec/services/barito_blueprint/processor_spec.rb

```diff
@@ -4,6 +4,9 @@ module BaritoBlueprint
   RSpec.describe Processor do
     before(:each) do
       @infrastructure = create(:infrastructure)
+      %w(consul yggdrasil).each do |n|
```

Comment:
we should make this as spec seeds

Reply:
done https://github.com/BaritoLog/BaritoMarket/commit/df772f2e655b3fee57db1553827078851fc4255f